src/main/java/edu/harvard/iq/dataverse/engine/command/impl/UpdateDatasetVersionCommand.java

```diff
@@ -280,13 +281,28 @@ public Dataset execute(CommandContext ctxt) throws CommandException {
                 AuthenticatedUser au = (AuthenticatedUser) getUser();
                 ctxt.datasetVersion().writeEditVersionLog(dvd, au);
             }
+            if ( theDataset != null ) {
```

Comment:
FWIW: I think this was unrelated/ related to other optimizations. If this PR is used for fixing the indirect list issue, this should be looked at /probably be removed/moved to it's own PR.